tests/data_context/fixtures/plugins/my_custom_expectations_store.py

```diff
@@ -0,0 +1,5 @@
+from great_expectations.data_context.store import ExpectationsStore
+
+
+class MyCustomExpectationsStore(ExpectationsStore):
+    pass
```

Comment:
Similar request as before : could we add a comment saying this class is used only for testing?